Software/vortex_ws/src/README.md

```diff
@@ -1 +1,28 @@
-All the ROS packages should be created in this fo;der
+## prerequisites¶
+#### Dependencies:
+- [Robot Operating System (ROS2)](https://index.ros.org/doc/ros2/Installation/Eloquent/Linux-Install-Debians/) (Install Eloquent as Debian Package).
+- [Colcon](https://index.ros.org/doc/ros2/Tutorials/Colcon-Tutorial/)
+#### You can  install Dependencies at the first time as below:
+
+* #### **System requirements¶**
+   **Target platforms for Eloquent Elusor are [(see REP 2000)](https://www.ros.org/reps/rep-2000.html):**
+    - **Tier 1: Ubuntu Linux - Bionic Beaver (18.04) 64-bit**
+    - **System setup**
+    - **Set locale**
+
+* Download **installSystem.sh** and **getRos2.sh** from **tools** > **scripts** > **ros_setup** folder
+* navigate their path then run the following commands.   
```

Comment:
Should be in a single step "Navigate to /tools/scripts/ros_setup then run the following scripts"

- For system dependences 
                         $./installSystem.sh

- For getting ROS 2 code && Installing dependencies using rosdep      
                           $./getRos2.sh